esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2214,8 +2229,37 @@ bool CFileSprayEx::onReplicate(IEspContext &context, IEspReplicate &req, IEspRep
     return true;
 }
 
+void CFileSprayEx::getDropZoneInfoByDestPlane(double clientVersion, const char* destGroup, const char* destFileIn, StringBuffer& destFileOut, StringBuffer& umask)
+{
+#ifdef _CONTAINERIZED
```

Comment:
probably applicable to bare-metal too eventually, perhaps should compile in, so checked.